### PR TITLE
Upgrade expat to version 2.4.1

### PR DESCRIPTION
**Pull request by 0xba1a**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Upgrade expat version to fix CVE-2019-15903

###### Change Log  

- Upgrade expat package version to 2.4.1

###### Does this affect the toolchain?  

**NO**

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2019-15903

###### Test Methodology

Build in local machine

### CODE REVIEW

**Review by MateuszMalisz — APPROVED** [review]

File: SPECS/expat/expat.spec [hunk 1]
```diff
@@ -1,71 +1,75 @@
...
+
+%ldconfig_scriptlets
 
-%post	-p /sbin/ldconfig
-%postun	-p /sbin/ldconfig
-%clean
-rm -rf %{buildroot}/*
 %files
 %defattr(-,root,root)
-%license COPYING
```
Comment (PawelWMS) [inline SPECS/expat/expat.spec:55]: I think there's no need to remove the license information from this package.

File: SPECS/expat/expat.spec [hunk 2]
```diff
@@ -1,71 +1,75 @@
...
 %files devel
 %{_includedir}/*
 %{_libdir}/pkgconfig/*
 %{_libdir}/libexpat.so
+%{_libdir}/cmake/expat-%{version}
 
 %files libs
-%{_libdir}/libexpat.so.*
+%license COPYING
+%{_libdir}/libexpat.so.1*
```
Comment (PawelWMS) [inline SPECS/expat/expat.spec:65]: Thanks for doing that!

**Review by PawelWMS — CHANGES_REQUESTED** [review]
This is a toolchain package, so a few more changes are required. I see we have already done this update in `dev`, so you might be able to take the changes from #1667.

**Review by MateuszMalisz — CHANGES_REQUESTED** [review]
Paweł raised a good point - please try to backport the existing changes from the dev instead.

**Comment by MateuszMalisz** [comment]
Another thing: please make sure to check whether the packages that depend on this package need to be rebuilt (I think python3's xml module might need a rebuild).

**Comment by jslobodzian** [comment]
expat is already a 2.4.6.  Closing this PR

**Pull request closed without merging**